GVRf/Framework/framework/src/main/java/org/gearvrf/GVRJassimpAdapter.java

```diff
@@ -134,6 +137,22 @@ public GVRMesh createMesh(GVRContext ctx, AiMesh aiMesh) {
             }
         }
 
+        // Vertex Colors
+        for(int c = 0; c < MAX_VERTEX_COLORS; c++) {
+            FloatBuffer fbuf = aiMesh.getColorBuffer(c);
+            if (fbuf != null) {
+                FloatBuffer coords = FloatBuffer.allocate(aiMesh.getNumVertices() * 4);
+                FloatBuffer source = aiMesh.getColorBuffer(c);
+                String name = "a_color";
```

Comment:
The full source looks like this (c is the color set index):
                String name = "a_color";

                if (c > 0) {
                    name += c;
                }